### PR TITLE
feat: add end to end Arm template run for kubetest

### PR DESCRIPTION
**Pull request by rjaini**
feat: add end to end Arm template run for kubetest

### CODE REVIEW

File: templates/k8s-test-infra/script.sh [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+	--acsengine-orchestratorRelease=${K8S_ORCHESTRATOR_VERSION} \
+	--acsengine-mastervmsize=${MASTER_SIZE} \
+	--acsengine-agentvmsize=${AGENT_SIZE} \
+	--acsengine-hyperkube=${BUILD_HYPERKUBE} \
+	--acsengine-public-key=${ACK_PUBLIC_KEY_PATH} \
+	--acsengine-ccm=${BUILD_CUSTOM_CLOUD_CONTROLLER} \
+	--acsengine-location=${REGION_NAME} \
+	--acsengine-template-url=${CLUSTER_DEFINITION} \
+	--acsengine-azure-env=${ENVIRONMENT_NAME} \
+	--acsengine-identity-system=${IDENTITY_TYPE} \
```
Comment (Bhuvaneswari-Santharam) [inline templates/k8s-test-infra/script.sh]: This parameter is not defined, is it supposed to be IDENTITY_SYSTEM?

File: templates/k8s-test-infra/script.sh [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+log_level -i "Constants"
+log_level -i "------------------------------------------------------------------------"
+
+export HOME=/root
+LOG_PATH="$HOME/logs"
+SECRET_PATH="$HOME/secret"
+ACK_PUBLIC_KEY_PATH="$SECRET_PATH/id_rsa.pub"
+AZURE_CREDS_PATH="$HOME/secret/azure_creds.toml"
+ENVIRONMENT_NAME=AzureStackCloud
+IDENTITY_SYSTEM_LOWER="azure_ad"
```
Comment (Bhuvaneswari-Santharam) [inline templates/k8s-test-infra/script.sh]: Why is IDENTITY_SYSTEM_LOWER hardcoded and not determined based on input IDENTITY_SYSTEM parameter?

Comment (rjaini) [reply templates/k8s-test-infra/script.sh]: Looks like I completely missed out with the identity type.

Comment (rjaini) [reply templates/k8s-test-infra/script.sh]: Update the PR with the fix.

File: templates/k8s-test-infra/script.sh (hunk 1 above)
Comment (rjaini) [reply templates/k8s-test-infra/script.sh]: Update the PR with the fix.

**Pull request merged**